README.md

```diff
@@ -9,13 +9,13 @@
 
 ## How it works
 
-"Scala Exercises" brings exercises for the Stdlib, Cats, Shapeless and many other great libraries for Scala to your browser. Offering hundreds of solvable exercises organized into several categories covering the basics of the Scala language and it's most important libraries.
+"Scala Exercises" brings exercises for Stdlib, Cats, Shapeless and many other great libraries for Scala to your browser. It offers hundreds of solvable exercises organized into several categories covering the basics of the Scala language and its most important libraries.
 
 - LEARN: Each category includes an explanation of the basics. Learn the concepts through simple code samples.
 
-- SOLVE: Each exercise is a unit test that must pass successfully, complete the exercise by filling in the blanks. Receive instant feedback as your answers are validated in real-time.
+- SOLVE: Each exercise is an unit test that must pass successfully, complete the exercise by filling in the blanks. Receive instant feedback as your answers are validated in real-time.
```

Comment:
"a unit test" is correct here. See the example with university here: https://dictionary.cambridge.org/grammar/british-grammar/determiners/a-an-and-the